config/vitest.config.ts

```diff
@@ -5,9 +5,13 @@ import { defineConfig, configDefaults } from 'vitest/config'
 import Vue from '@vitejs/plugin-vue'
 
 export default defineConfig({
-    plugins: [
-        Vue(),
-    ],
+    plugins: [Vue({
+        template: {
+            transformAssetUrls: {
+                includeAbsolute: false
```

Comment:
Prevents issue with compilation where <img src="/absolute.jpg" /> is failed to be found at build time